src/object/cli_ec.c

```diff
@@ -1664,7 +1664,7 @@ obj_ec_encode(struct obj_reasb_req *reasb_req)
 	uint32_t	i;
 	int		rc;
 
-	if (reasb_req->orr_usgls == NULL) /* punch case */
+	if (reasb_req->orr_processed_sgls == NULL) /* punch case */
```

Comment:
why need change this line?
"reasb_req->orr_usgls" is original user input sgl right?

Reply:
I think now it works like this: user sgls, and then processed sgls(trimmed some zero length but, allocate and merge sgls buffer if necessary), then reasb actually use processed sgls(instead of user sgls). so I changed from orr_usr_sgls to orr_processed_sgls.

Reply:
orr_usgls is nothing changed user input sgls, looks some misunderstanding for either you or me